pkg/imgpkg/v1/describe.go

```diff
@@ -187,5 +212,32 @@ func (r *refWithDescription) describeBundleRec(visitedImgs map[string]refWithDes
 		}
 	}
 
-	return desc.bundle
+	return desc.bundle, nil
+}
+
+func getImageLayersInfo(image string) ([]Layers, error) {
+	layers := []Layers{}
+	parsedImgRef, err := regname.ParseReference(image, regname.WeakValidation)
+	if err != nil {
+		return nil, fmt.Errorf("Error: %s in parsing image %s", err.Error(), image)
```

Comment:
nit: I think the error would already contain "Error", so just "Parsing image: err" something like this should be good?

Reply:
Seems error returned from  [ParseReference](https://github.com/google/go-containerregistry/blob/main/pkg/name/ref.go#L47) does not contain error as substring. I think this should be fine?